docs/UserGuide.adoc

```diff
@@ -59,8 +66,44 @@ Write CRUD stuff
 === Pet (Fyonn)
 Write pet
 
-=== Pomodoro (Hardy)
-Write pomodoro
+[[Pomodoro]]
+=== Pomodoro
+_This section describes the Pomodoro feature in detail and contains all commands you can use to take advantage of it_
+
+==== What's Pomodoro?
+In the late 1980s, a gentleman named Francesco Cirillo devised a time management method called the Pomodoro Technique. Essentially, a single cycle consists of two parts, 25 minutes of work, followed by a 5 minutes break. This cycle repeats for as long as you want to get work done.
+
+==== How do you use it?
+
+===== Command: `pom`
```

Comment:
it would be good if you can add a sort of scenario/preface to this section. For example: Trying to prevent burnout? BB productive offers you the option to try out the pomodoro tenchinique! The pomodoro technique breaks down work into manageable intervals spaced out with short breaks so as to not tired you out. To get started, follow the instructions below:

INSTRUCTIONS:
step 1......

you might also want to add screenshots of success messages (eg. after timer ends) to ensure the reader.

Reply:
Will do!